### PR TITLE
Fixes #37082 - Show 'Awaiting start' as status

### PR DESCRIPTION
**Pull request by adamruzicka**
of template invocations when the backing task is not yet created.

image

### CODE REVIEW

**Review by lhellebr — APPROVED** [review]
I think `pending` and `awaiting start` may be confused for each other but I don't have any better word proposal. LGTM.

File: app/helpers/remote_execution_helper.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def host_counter(label, count)
   end
 
   def template_invocation_status(task, parent_task)
-    return(parent_task.result == 'cancelled' ? _('cancelled') : 'N/A') if task.nil?
+    return(parent_task.result == 'cancelled' ? _('cancelled') : _('Awaiting start')) if task.nil?
```
Comment (lhellebr) [inline app/helpers/remote_execution_helper.rb:21]: Should a translation be added in a separate PR?

Comment (adamruzicka) [reply app/helpers/remote_execution_helper.rb:21]: Yes, strings are extracted for translation separately.
